src/common/options/rwl-replica.yaml.in

```diff
@@ -0,0 +1,84 @@
+# -*- mode: YAML -*-
+---
+
+options:
+- name: rwl_replica_addr
```

Comment:
For multi rwl_replica_damon, is it work?  I think we can use  start rwl_replica_daemon w/ rdma_addr + port.

Reply:
Yes, they can work. For example:
```
ceph-rwl-replica -c ceph.conf --log-file out/replica1.log --debug-rwl-replica 20 --rwl_replica_addr 10.0.0.20:9001 --rwl_replica_path /mnt/pmem/replica1 --rwl_replica_size $cache_size
ceph-rwl-replica -c ceph.conf --log-file out/replica2.log --debug-rwl-replica 20 --rwl_replica_addr 192.168.0.20:9002 --rwl_replica_path /mnt/pmem/replica2 --rwl_replica_size $cache_size
ceph-rwl-replica -c ceph.conf --log-file out/replica3.log --debug-rwl-replica 20 --rwl_replica_addr 192.168.0.20:9003 --rwl_replica_path /mnt/pmem/replica3 --rwl_replica_size $cache_size

```